frontend/appflowy_flutter/lib/window/app_window_listner.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:convert';
+
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:window_manager/window_manager.dart';
+
+class AppWindoeListner with WindowListener {
+  void initListner() {
+    windowManager.addListener(this);
+  }
+
+  @override
+  Future<void> onWindowResize() async {
+    final currentWindowSize = await WindowManager.instance.getSize();
+    final windowSize = {
+      'height':
+          currentWindowSize.height < 400.0 ? 400.0 : currentWindowSize.height,
+      'width':
+          currentWindowSize.width < 600.0 ? 600.0 : currentWindowSize.width,
+    };
+    final prefs = await SharedPreferences.getInstance();
```

Comment:
You can make it so `getInstance` is only called once per `AppWindowListener` instance.